lang/cs/Org.Apache.REEF.Client.Tests/packages.config

```diff
@@ -18,7 +18,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
-  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" />
```

Comment:
Is there a way for us to have the versions and such specified on the solution level? Much like we do in maven in the `dependencyManagement` section of the parent pom?